.php-cs-fixer.cache

```diff
@@ -1 +1 @@
1173917e","app\/Console\/Kernel.php":"8794bf696de8346c50cd9cf9dc0a6465","app\/Exceptions\/Handler.php":"883d928c9ddacb4e1841d6cae92c1762","app\/Providers\/AuthServiceProvider.php":"26b2ed4a6aca63b3a8c335441f2aea3f","app\/Providers\/RouteServiceProvider.php":"b1b38f2a1757ddbd2e8a1e021dadf2fb","app\/Providers\/AppServiceProvider.php":"c0094f35cad6bba7636002a62186dae4","app\/Providers\/EventServiceProvider.php":"be5c3b6feade2c917f7ac92d3769cd4d","app\/Providers\/BroadcastServiceProvider.php":"2ec8fd9c95c859ab7299b84344e1e854","app\/Models\/User.php":"16880435a038d81c20ee7884b0b27307","config\/app.php":"735ca85495c078722e32536e2f0ef882","config\/logging.php":"d4900907cd3456e3196850266b5982ba","config\/cors.php":"7e46f4152490ce5bf22fe42d2b57a63a","config\/queue.php":"104211dcfd96874df6661a47da94c3b7","config\/hashing.php":"7f0c219eabd21347095bc9263dbfd882","config\/database.php":"7e1fe950dc6e5ae47d7ff3ce5458aa64","config\/mail.php":"9818b80eda9ee8c49d4406128a4cdb67","config\/sanctum.php":"cbb6b4208f604062f892f7cbd1981d70","config\/scribe.php":"2d20f1c6adb47ce927772e6b6b2b64be","config\/filesystems.php":"4fd376ad6cba1ff7a09c0bd562d0bb21","config\/view.php":"24e00ccca17634d028fbbfdafd9e7461","config\/services.php":"1146126dd89fa5a45a7b8fe980859d20","config\/cache.php":"0342238d20a6ea9523947d70c037bf83","config\/auth.php":"00777931ac1cdee35ad18d8256395a99","config\/broadcasting.php":"d95677f597c30e32af4e6b1e67d33fbb","config\/session.php":"3a7383033e5590fed7fbab85f2ce924e","database\/factories\/UserFactory.php":"8883ee068fed670be1b9cb11ba7730e6","database\/seeders\/DatabaseSeeder.php":"bb825063b4bc94f2d61fed1e9c07250b","database\/migrations\/2019_08_19_000000_create_failed_jobs_table.php":"7bb8cef34acf8036dc5c18c2f0be256c","database\/migrations\/2014_10_12_100000_create_password_reset_tokens_table.php":"da2270e80e1dad940468e2a92fe6930a","database\/migrations\/2014_10_12_000000_create_users_table.php":"434c2c415172e0e856f7555f309111fe","database\/migrations\/2019_12_14_000001_create_personal_access_tokens_table.php":"541c0bd2df22e987ec40fa653c20a56a","routes\/console.php":"62bfb59371c073b4ec966528fdd9c3f9","routes\/channels.php":"5edf0b9faf77f23d5bb0474740fe651d","routes\/web.php":"0ccb0f0732887a32e8c909c34df67a9f","routes\/api.php":"3ac69399a0d70d6aa7eba960f51b4215","tests\/Unit\/ExampleTest.php":"3f679e9508c834f0ad95943455fcd836","tests\/TestCase.php":"93519c98470fc8240aed892b40c5a9fc","tests\/Feature\/ExampleTest.php":"0f4d91c9c5bbcd4e2f46cf885acb7fb6","tests\/CreatesApplication.php":"b8e30e361b5218089bf3d1f2a1059a11","app\/Http\/Requests\/Auth\/LoginRequest.php":"95e2a0d1afbe6459462ab0a6b48e4ce0","app\/Http\/Requests\/Auth\/RegisterRequest.php":"b421ddc2ac0e8c4e32f8781849d23bfb","app\/Http\/Controllers\/MainPlatform\/DashboardController.php":"ef516a11d27451bf94a77fb17fa0f5aa","app\/Http\/Controllers\/Auth\/RegisterController.php":"cce7410842df9e37417f2eace1d19393","app\/Http\/Controllers\/Auth\/LoginController.php":"3fa1264a9fbd09bc85c9477275858c81"}}
```

Comment:
You can add this file to .gitignore and delete it from the repository completely.

Reply:
Everything should be done by tonight.

Reply:
Done.